iroh-gateway/src/core.rs

```diff
@@ -80,10 +80,10 @@ impl Handler {
             download,
         };
         match req.format {
-            ResponseFormat::Raw => Handler::serve_raw(&req, client, &mut headers).await,
-            ResponseFormat::Car => Handler::serve_car(&req, client, &mut headers).await,
-            ResponseFormat::HTML => Handler::serve_html(&req, client, &mut headers).await,
-            ResponseFormat::FS => Handler::serve_fs(&req, client, &mut headers).await,
+            ResponseFormat::Raw => Core::serve_raw(&req, client, &mut headers).await,
```

Comment:
no need for these methods to be attached to Core, they could be just free standing